test/unit/migrations/multiple_ldap_hostnames_migration_test.rb

```diff
@@ -0,0 +1,104 @@
+# encoding: utf-8
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+require 'test_helper'
+migration_file_name = Dir[Rails.root.join('db/migrate/20171010071016_multiple_ldap_hostnames.rb')].first
+require migration_file_name
+
+class MultipleLdapHostnamesMigrationTest < ActiveSupport::TestCase
+
+  describe '#up' do
+    test 'does not change type if hostname is an array' do
+      MultipleLdapHostnames.any_instance.expects(:hostname)
+               .returns(['yourdomain.com'])
+
+      assert_nil migration.up
```

Comment:
do not rely on return value of up method. test behavior with expects for example.